docs/github/v2/15-deployment/ios.md

```diff
@@ -211,54 +211,54 @@ buildForiOSPlatform:
         name: build-iOS
         path: build/iOS
 
-  releaseToAppStore:
-    name: Release to the App Store
-    runs-on: macos-latest
-    needs: buildForiOSPlatform
-    env:
-      APPLE_CONNECT_EMAIL: ${{ secrets.APPLE_CONNECT_EMAIL }}
-      APPLE_DEVELOPER_EMAIL: ${{ secrets.APPLE_DEVELOPER_EMAIL }}
-      APPLE_TEAM_ID: ${{ secrets.APPLE_TEAM_ID }}
-      APPLE_TEAM_NAME: ${{ secrets.APPLE_TEAM_NAME }}
-      MATCH_URL: ${{ secrets.MATCH_URL }}
-      GIT_TOKEN: ${{ secrets.GIT_TOKEN }}
-      MATCH_PASSWORD: ${{ secrets.MATCH_PASSWORD }}
-      APPSTORE_KEY_ID: ${{ secrets.APPSTORE_KEY_ID }}
-      APPSTORE_ISSUER_ID: ${{ secrets.APPSTORE_ISSUER_ID }}
-      APPSTORE_P8: ${{ secrets. APPSTORE_P8 }}
```

Comment:
The indentation was already correct. Now you can no longer copy paste it without having to use tabs (and not everyone has their editor set to use spaces instead of tabs.